.vscode/settings.json

```diff
@@ -22,7 +22,7 @@
   ],
   "i18n-ally.preferredDelimiter": "_",
   "i18n-ally.sortKeys": true,
-  "i18n-ally.sourceLanguage": "en-US",
+  "i18n-ally.sourceLanguage": "es-419",
```

Comment:
This should be reverted, no?

Reply:
Should I add this file to gitignore?

Reply:
@userquin do you know what is best practice here?

Reply:
I don't use VSCode, I don't add IntelliJ stuff to my GIT, I think it should be excluded

Reply:
I reverted it, and I will keep it untouched until Core team decides how to proceed for this.